test/Controllers/ServicesTest.php

```diff
@@ -44,11 +43,10 @@ private function serviceBody($type) {
             case 'create':
                 return new CreateServiceBody(
                     "description", // REQUIRED
-                    15, // duration REQUIRED XXX 0 not accepted but NULL turns into 0
-                    // XXX No max_duration in SDK
```

Comment:
Wait, there's more! This comment also signaled another posible bug. Seem the SDK is not up to date with API? `max_duration` isn't being returned. FYI. Thanks